.github/workflows/release.yml

```diff
@@ -31,10 +36,8 @@ jobs:
           ref: "main"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Build and publish to PyPi
-        env:
-          POETRY_PYPI_TOKEN_PYPI: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
Once the new release process confirmed, don't forget to remove the secret.
By the way, I really like this news trusted publishing :)

Reply:
Good one!